extensions/sql-database-projects/src/test/project.test.ts

```diff
@@ -707,26 +707,117 @@ describe('Project: sqlproj content operations', function (): void {
 	it('Project entry relative path should not change after round-trip', async function (): Promise<void> {
```

Comment:
does "round-trip" here mean closing and reopening a project in ADS? Usually "round-trip" is used as in round-tripping between ADS and SSDT

Reply:
Yeah, here it refers to save and reload, as I hit a bug there that I've addressed previously. I guess it depends on how long is your trip :)

Do you have better terminology for save and load? Maybe `after reload`?

Reply:
`after reload` sounds good!